core/modules/layout/includes/layout.class.inc

```diff
@@ -691,11 +691,26 @@ class Layout {
             $new_positions[$new_region_name][] = $uuid;
           }
           else {
-            unset($this->content[$uuid]);
+            $orphaned_blocks[] = $uuid;
           }
         }
       }
     }
+    if (!empty($orphaned_blocks)) {
+      // Assign orphan blocks to the default or last region,
```

Comment:
Extremely finicky and unimportant, but why line break at 60 characters?

Reply:
A comma seemed like a good place to pause and start a new line.

Reply:
We don't do that though (emphasis mine): https://api.backdropcms.org/doc-standards#general

> - Lines containing comments (including docblocks) must wrap **as close to 80 characters as possible** without going over